app/controllers/api/account.php

```diff
@@ -166,6 +166,100 @@
     $response->dynamic($session, Response::MODEL_SESSION);
 };
 
+$sendMFAAlert = function (Document $project, Document $user, Locale $locale, Mail $queueForMails, string $type, string $redirect) {
```

Comment:
If we only do this in one route, let's do it inline instead of declaring a global function var